node-tests/nodetest-runner.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var glob = require('glob');
+var Mocha = require('mocha');
+var Promise = require('ember-cli/lib/ext/promise');
```

Comment:
I think we can change it to just RSVP. I'm fairly certain that these specific cases don't require any of the added functionality provided by ext promise (@stefanpenner please correct me if I'm wrong).

I'll change it upstream in the ember-cli-blueprint-test-helpers blueprint.